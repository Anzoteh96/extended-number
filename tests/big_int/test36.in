-9511194968862191065515481860367904252740742341215264429858852976691513795978270762331213569373453683539212915
-881527635091777971187852937027861253086617759174540327930717265706063715829922911350590883315582949328162126
