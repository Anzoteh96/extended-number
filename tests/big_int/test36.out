Enter two numbers n and m:
n is -9511194968862191065515481860367904252740742341215264429858852976691513795978270762331213569373453683539212915
m is -881527635091777971187852937027861253086617759174540327930717265706063715829922911350590883315582949328162126
n + m is -10392722603953969036703334797395765505827360100389804757789570242397577511808193673681804452689036632867375041
n - m is -8629667333770413094327628923340042999654124582040724101928135710985450080148347850980622686057870734211050789
n * m is 8384381207797904108716278764408237810152738139160677588607352533857884663355548518633905922424388411982256265320016100054529959445273828731229963632535392959954866479031565399158549879153142846712562607345649553057290
n / m is 10
n % m is -695918617944411353636952490089291721874564749469861150551680319630876637679041648825304736217624190257591655
